.ansible-lint

```diff
@@ -1,2 +1,9 @@
+extends: default
+rules:
```

Comment:
This looks a lot like a yamllint config. Who is reading and processing this part (both `extends` and `rules`)?

Reply:
@felixfontein Thanks, that was added by mistake, I removed it.